src/main/java/org/jabref/gui/externalfiles/DuplicateDecisionResult.java

```diff
@@ -3,14 +3,9 @@
 import org.jabref.gui.duplicationFinder.DuplicateResolverDialog;
 import org.jabref.model.entry.BibEntry;
 
-public class DuplicateDecisionResult {
-    private final DuplicateResolverDialog.DuplicateResolverResult decision;
-    private final BibEntry mergedEntry;
-
-    public DuplicateDecisionResult(DuplicateResolverDialog.DuplicateResolverResult decision, BibEntry mergedEntry) {
-        this.decision = decision;
-        this.mergedEntry = mergedEntry;
-    }
+public record DuplicateDecisionResult(
+        DuplicateResolverDialog.DuplicateResolverResult decision,
+        BibEntry mergedEntry) {
```

Comment:
Note that the IntelliJ refactoring did more. It renamed the methods, so that explicit `getDecision()` and `getMergedEntry()` are not needed any more. One can now just call `decision()` and `mergedEntry()`. Please follow that suggestion.